Engine/ac/mouse.cpp

```diff
@@ -535,8 +581,10 @@ void RegisterMouseAPI()
     ccAddExternalStaticFunction("Mouse::EnableMode^1",              Sc_enable_cursor_mode);
     ccAddExternalStaticFunction("Mouse::GetModeGraphic^1",          Sc_Mouse_GetModeGraphic);
     ccAddExternalStaticFunction("Mouse::IsButtonDown^1",            Sc_IsButtonDown);
+    ccAddExternalStaticFunction("Mouse::IsModeEnabled^1",           Sc_IsModeEnabled);
     ccAddExternalStaticFunction("Mouse::SaveCursorUntilItLeaves^0", Sc_SaveCursorForLocationChange);
     ccAddExternalStaticFunction("Mouse::SelectNextMode^0",          Sc_SetNextCursor);
+    ccAddExternalStaticFunction("Mouse::SelectPreviousMode^0",      Sc_SetPreviousCursor);
```

Comment:
This is a legacy issue, in AGS "cursors" and "modes" are strictly tied components of same entity (the graphical and logical ones), the type is even called "CursorMode", which can refer both to chosen interaction mode and its cursor.
The cursor has sprite attached to it, which may be swapped, but cursor is always same for the given mode, so in practice saying "next cursor" is same as "next mode".